server/sonar-server-common/src/main/java/org/sonar/server/webhook/WebHooksImpl.java

```diff
@@ -122,7 +122,7 @@ private void sendProjectAnalysisUpdateImpl(Analysis analysis, Supplier<WebhookPa
   private static void log(WebhookDelivery delivery) {
     Optional<String> error = delivery.getErrorMessage();
     if (error.isPresent()) {
-      LOGGER.debug("Failed to send webhook '{}' | url={} | message={}",
+      LOGGER.error("Failed to send webhook '{}' | url={} | message={}",
```

Comment:
@anurag-ar it's better to use `warn` not `error`. 

e.g.
https://sematext.com/blog/logging-levels/
https://stackoverflow.com/a/5278006/427002